geotrek/common/management/commands/sync_rando.py

```diff
@@ -263,8 +264,10 @@ def sync_file(self, lang, name, src_root, url, zipfile=None):
         if self.verbosity == 2:
             self.stdout.write("{lang} {url}/{name} copied".format(lang=lang, url=url, name=name))
 
-    def sync_static_file(self, lang, name):
-        self.sync_file(lang, name, settings.STATIC_ROOT, settings.STATIC_URL)
+    def sync_static_file(self, lang, name, zipfile=None):
+        file_path = find(name)
+        file_dir, file_name = os.path.split(file_path)
```

Comment:
The parameter name is sowewhat misnamed. It can contains a directory element before the file name. So this change is not equivalent and creates bugs when using data in Geotrek-rando. Moreover it makes the code more tricky.